### PR TITLE
Rob/Displays Seller data from back end

### PR DESCRIPTION
**Pull request by ArtyEmsee**
Pulls data from back end and displays on seller page.

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -37,7 +16,7 @@ class App extends React.Component<
           {
             // TO DO: add router config for this route
           }
-          <Route path="/story/:id" component={MerchantPage} />
+          <Route path="/story/:id" component={SellerPage} />
```
Comment (jtmckibb) [inline src/components/App/App.tsx]: Is it possible to do :id/story? It looks funny in the URL this way, and if we're linking people here I don't want the URL to change

Comment (jtmckibb) [reply src/components/App/App.tsx]: Actually, can we change it to just :id ? The other tabs can have :id/menu and :id/photos

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 2]
```diff
@@ -54,22 +52,25 @@ class OwnerPanel extends React.Component<Props, State> {
...
-            >
-              {' '}
+        {this.props.targetAmount && this.props.amountRaised && (
+          <div className={styles.progressContainer}>
+            <div className={classnames(styles.progressBar, 'progress-bar')}>
+              <div
+                className={styles.myBar}
+                style={{
+                  width: `${
+                    (this.props.amountRaised / this.props.targetAmount) * 100
```
Comment (jtmckibb) [inline src/components/OwnerPanel/OwnerPanel.tsx:62]: nice

File: src/components/SellerPage/SellerPage.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    const result = await getSeller();
+    setSeller(result.data);
+  };
+
+  // TO DO: handle actual null states and loading
```
Comment (jtmckibb) [inline src/components/SellerPage/SellerPage.tsx:26]: @kevinjtsai FYI we'd need a null state page. Maybe we can ask Jess or Holly to render something for us?

File: src/components/StoreDetails/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Props } from './StoreDetails';
+// import { Props } from './StoreDetails';
```
Comment (jtmckibb) [inline src/components/StoreDetails/index.ts:1]: delete?

File: src/components/StoreInfo/StoreInfo.tsx [hunk 5]
```diff
@@ -1,46 +1,41 @@
...
-      <div className={styles.nationality}>{nationality}</div>
+      <div className={styles.nationality}>{cuisineName}</div>
       <p>
-        <div className={styles.address}>{address.line1}</div>
-        <div className={styles.address}>{address.line2}</div>
+        {addresses &&
+          addresses.map((address: Address) => (
+            <>
+              <div className={styles.address}>{address.address1}</div>
+              <div className={styles.address}>{address.address1}</div>
```
Comment (jtmckibb) [inline src/components/StoreInfo/StoreInfo.tsx:29]: should be address2

File: src/components/StoreInfo/StoreInfo.tsx [hunk 6]
```diff
@@ -1,46 +1,41 @@
...
-    className: styles.storeDetails,
-  };
   return (
     <section className={classnames(styles.container, className)}>
-      <div className={styles.nationality}>{nationality}</div>
+      <div className={styles.nationality}>{cuisineName}</div>
       <p>
-        <div className={styles.address}>{address.line1}</div>
-        <div className={styles.address}>{address.line2}</div>
+        {addresses &&
```
Comment (jtmckibb) [inline src/components/StoreInfo/StoreInfo.tsx:25]: @kevinjtsai I added support of multiple addresses in the case that they have them. Could we maybe see how design would like for this to look?

File: src/components/StoreInfo/StoreInfo.tsx [hunk 7]
```diff
@@ -1,46 +1,41 @@
...
+          addresses.map((address: Address) => (
+            <>
+              <div className={styles.address}>{address.address1}</div>
+              <div className={styles.address}>{address.address1}</div>
+              <div className={styles.address}>{address.city}</div>
+              <div className={styles.address}>{address.state}</div>
+              <div className={styles.address}>{address.zip_code}</div>
+            </>
+          ))}
         <div className={styles.address}>{phoneNumber}</div>
```
Comment (jtmckibb) [inline src/components/StoreInfo/StoreInfo.tsx:35]: Merp looking at this code made me realize something — I think that address should be renamed to 'locations". I'm also going to move this phone number to the address field. Sorry for the churn, but do you mind updating this? I'll update the backend in parallel

File: src/utilities/api/endpoints.ts [hunk 8]
```diff
@@ -1,5 +1,5 @@
 const baseUrl = process.env.REACT_APP_API_ENDPOINT!;
 
 export const sellers = baseUrl + 'sellers';
-export const seller = baseUrl + 'sellers/test_kitchen';
+export const seller = baseUrl + 'sellers/dian-kitchen';
```
Comment (jtmckibb) [inline src/utilities/api/endpoints.ts]: This shouldn't be static should it be? We can pull this from the server

File: src/components/SellerPage/sample-merchant.ts [hunk 9]
```diff
@@ -1,4 +1,4 @@
-import avatar from './images/female.svg'
+import avatar from './images/female.svg';
```
Comment (jtmckibb) [inline src/components/SellerPage/sample-merchant.ts:1]: The URL for the owner photo should come from `owner_image_url`

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 10]
```diff
@@ -54,22 +52,25 @@ class OwnerPanel extends React.Component<Props, State> {
...
             </div>
           </div>
-          {/* pass props down here*/}
-          <div>
-            ${this.state.current} of ${this.state.need}
-          </div>
-        </div>
+        )}
 
         <div className={styles.buttonContainer}>
```
Comment (jtmckibb) [inline src/components/OwnerPanel/OwnerPanel.tsx:75]: The buttons should depend on `accept_donations` and `sell_gift_card` response

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 11]
```diff
@@ -7,11 +7,11 @@ interface Props {
   ownerName: string;
```
Comment (jtmckibb) [inline src/components/OwnerPanel/OwnerPanel.tsx:7]: Are we using ownerName from teh response too?

File: src/utilities/api/types.ts [hunk 12]
```diff
@@ -6,12 +6,25 @@ export type PaymentParams = {
...
+  className: '';
+  cuisineName: string;
+  email?: string;
+  name: string;
+  
+  //TO DO: why is this necessary here
+  showPayModal?: boolean;
+  summary: string;
+  story: string;
 };
```
Comment (jtmckibb) [inline src/utilities/api/types.ts:30]: Yeah it looks like we need ownerName, acceptsDonations, sellsGiftCards, ownerImageUrl

File: src/components/SellerPage/sample-merchant.ts (hunk 9 above)
Comment (ArtyEmsee) [reply src/components/SellerPage/sample-merchant.ts:1]: that's sample data though, the whole thing should go. I was gonna work that out in a owner pic follow up where we standardize default pics.

Comment (jtmckibb) [reply src/components/SellerPage/sample-merchant.ts:1]: Sounds good!

**Comment by jtmckibb** [comment]
@ath-chen @ArtyEmsee still review this PR! I'll address any comments as a follow up

**Pull request merged**